tests/_support/Mage/Mage_Payment_Model_Method_Abstract.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+class Mage_Payment_Model_Method_Abstract
+{
+    public function getInfoInstance()
+    {
+        return new Mage_Payment_Model_Method_Abstract();
+    }
+
+    public function getCcOwner()
```

Comment:
O que é `Cc` ?

Reply:
**CrediCard**.
É um método do próprio Magento :smile:

Reply:
Podemos criar um método chamado `getCredicardOwner` que `encapsula` este método próprio do `Magento`

Reply:
Acredito não ser necessário.
Temos uma documentação fértil da comunidade Magento e ela é utilizada como base em todas as nossas implementações.
Lembrando que esse arquivo é apenas um Helper para auxiliar nosso framework a executar se que seja necessário uma aplicação Magento em execução.

Estou resolvendo esse comentário. mas qualquer coisa temos uma seção de [Issues](https://github.com/vindi/vindi-magento/issues/new/choose) que pode ser utilizada para reportar esse tipo de situação :+1:

Reply:
Não estou entendo este fluxo de `conversation` que você está adotando, eu adiciono um comentário, você responde e já marca a conversa como resolvida antes de eu ler ou adicionar comentários adicionais?

Acredito que não é por que a comunidade adota nomes poucos descritivos que você deva seguir o mesmo caminho e repetir o erro, quanto mais claro for o seu código será melhor para você e para o próximo.

Muito obrigado pela sugestão de abertura de `Issues`, mas acredito que aqui seja o momento e local para discurtimos sobre o codebase que está em revisão, caso após a conclusão da discussão, vejamos que é necessária a abertura de uma `Issue` podemos ir para essa abordagem.

Reply:
Conforme conversamos, estamos avaliando a possibilidade de adotar boas práticas e possíveis `refactorings` no módulo. Isso poderá ser implementado em novos PRs